ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
I think you mean - is there a bug waiting to happen?

If the programmer has not turned the motor on yet, then this function will wait quite some time!